### DISCUSSION TITLE
Is it possible to see a log of all past sessions on different devices? [Help Wanted]

### DISCUSSION THREAD

**Discussion by StGabriels** [standardnotes/forum#2745] [Suggestions]
I like the feature of "active sessions" but would like to have a log of all past sessions which shows which device was in use each time. Is this information already available in some hidden log file, and if not, is there a way to create such a log? 

I'm not a real tech geek so simple directions are much appreciated. Thanks!

**Comment by JaspalSuri** [comment]
Hi @StGabriels, this currently isn't possible.
